uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/staff-leave-report.component.js

```diff
@@ -311,6 +319,20 @@ define([
         });
     }
 
+    /**
+     * Loads leave requests and populates them to the given section
+     *
+     * @param  {String} section
+     * @param  {Object} params extra parameters for the leave requests search
+     * @return {Promise}
+     */
+    function loadLeaveRequests (section, params) {
+      return fetchLeaveRequests(params)
```

Comment:
Ok, so now we have a `loadLeaveRequests` and a `fetchLeaveRequests` in the same file.

I propose this (you can fix it any way you like):

- loadLeaveRequests should be renamed to loadLeaveRequestsForSection
- fetchLeaveRequests should be renamed to loadLeaveRequests